core/templates/dev/head/pages/about/about.html

```diff
@@ -44,7 +44,6 @@
               <div class="oppia-about-text-align">
                 <em>Oppia "O-pee-yah" (Finnish) - "to learn"</em>
               </div>
-              
```

Comment:
Please remove this change. Even though it's very minor, but it would certainly be better if this didn't go in!